docs/howto/interop/index.md

```diff
@@ -0,0 +1,295 @@
+    that tests all three properties listed above:
+
+    ```cpp
+    template <typename NholthausType, typename AuUnit>
+    void expect_equivalent(QuantityMaker<AuUnit> expected_au_unit) {
+        const auto original = NholthausType{1.2};
+        const auto equivalent_to_expected_au_unit = QuantityEquivalent(expected_au_unit(1.2));
+
+        // Check that an `as_quantity` mapping _exists_ for this nholthaus type, and that its result
+        // is quantity-equivalent to the given QuantityMaker applied to the same underlying value.
+        const auto converted_to_quantity = as_quantity(original);
+        EXPECT_THAT(converted_to_quantity, equivalent_to_expected_au_unit);
+
+        // Check that this nholthaus type is _implicitly_ convertible to its equivalent type, and
+        // that again, the result is quantity-equivalent to the given QuantityMaker applied to the
+        // same underlying value.
+        const decltype(converted_to_quantity) implicitly_converted_to_quantity = original;
+        EXPECT_THAT(implicitly_converted_to_quantity, equivalent_to_expected_au_unit);
+
+        // Check that the equivalent Quantity type can be _implicitly_ converted back to the
+        // original nholthaus type, and that this round trip is the identity.
+        const NholthausType round_trip = implicitly_converted_to_quantity;
+        EXPECT_EQ(round_trip, original);
+    }
+    ```
+
+    (In the above, note that `QuantityEquivalent` is a utility from the Au library.  It's found in
+    `"au/testing.hh"`, which is available in our [full installation](../../install.md#full) but not
+    in our [single-file installation](../../install.md#single-file).  This matcher tests an object
+    against the value in its constructor (here, `expected_au_unit(1.2)`).  It makes sure that their
+    _types_ are [quantity-equivalent](../../reference/quantity.md#are-quantity-types-equivalent),
+    and their _values_ are equal.)
+
+    Next, we write test cases which take advantage of this utility, such as [this
+    group](https://github.com/aurora-opensource/au/blob/3fa22a2/compatibility/nholthaus_units_test.cc#L74-L82).
+
+    ```cpp
+    TEST(NholthausTypes, MapsBaseUnitsOntoCorrectAuQuantityTypes) {
+        expect_equivalent<::units::length::meter_t>(meters);
+        expect_equivalent<::units::mass::kilogram_t>(kilo(grams));
+        expect_equivalent<::units::time::second_t>(seconds);
+        expect_equivalent<::units::angle::radian_t>(radians);
+        expect_equivalent<::units::current::ampere_t>(amperes);
+        expect_equivalent<::units::temperature::kelvin_t>(kelvins);
+        expect_equivalent<::units::data::byte_t>(bytes);
+    }
+    ```
+
+    Note how the utility makes these very easy to read.
+
+### Implement the Definitions
+
+The general strategy for writing an implementation that can make these unit tests pass includes
+these steps.
```

Comment:
Read the above sentence aloud -- it's a mouthful. How about:

"Generally, writing an implementation that passes these unit tests includes:

1. Figuring out...
2. Writing...
3. Writing..."

...It changes the imperative verbs in the list to gerunds, which contradicts my normal advice, but for the sake of making the lead-in sentence a little clearer, I think it's worth it.

Reply:
Done.